src/uct/tcp/tcp_net.c

```diff
@@ -184,38 +214,67 @@ ucs_status_t uct_tcp_netif_is_default(const char *if_name, int *result_p)
     return UCS_OK;
 }
 
-static ucs_status_t uct_tcp_do_io(int fd, void *data, size_t *length_p,
-                                  uct_tcp_io_func_t io_func, const char *name)
+int uct_tcp_sockaddr_cmp(const struct sockaddr *sa1,
```

Comment:
why do you need it?

Reply:
it would be needed to compare addresses for hashing/resolving simultaneous connections (peer will smaller addresses will has a priority)

Reply:
so it should be added to the PR which adds hash table